x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/enrich/AbstractLookupService.java

```diff
@@ -143,7 +144,7 @@ abstract class AbstractLookupService<R extends AbstractLookupService.Request, T
 
     AbstractLookupService(
         String actionName,
-        String privilegeName,
+        @Nullable String privilegeName,
```

Comment:
This is a very plain change to make it work, but I think we don't need much more for now. Easy to change later if we want something more complex